### PR TITLE
add distribution `laplace`

### PR DESCRIPTION
**Pull request by YoshikawaMasashi**
split from #1177

### CODE REVIEW

File: cupy/random/kernels.py [hunk 1]
```diff
@@ -0,0 +1,27 @@
+from cupy import core
+
+_gumbel_kernel = None
+_laplace_kernel = None
+
+
+def _get_gumbel_kernel():
+    global _gumbel_kernel
+    if _gumbel_kernel is None:
+        _gumbel_kernel = core.ElementwiseKernel(
```
Comment (okuta) [inline cupy/random/kernels.py]: Please assign in global space.
Creating `ElementwiseKernel` kernel takes only a little time.

File: cupy/random/kernels.py [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+            'y = loc - log(-log(1 - x)) * scale',
+            'gumbel_kernel'
+        )
+    return _gumbel_kernel
+
+
+def _get_laplace_kernel():
+    global _laplace_kernel
+    if _laplace_kernel is None:
+        _laplace_kernel = core.ElementwiseKernel(
```
Comment (okuta) [inline cupy/random/kernels.py]: ditto

**Comment by okuta** [comment]
jenkins, test this please.

**Comment by YoshikawaMasashi** [comment]
I fixed.

**Comment by okuta** [comment]
OK.
And, please fix rst file to add functions to document.

**Comment by okuta** [comment]
Cloud you rename `kernel.py` to `_kernel.py`?
It is internal module.

LGTM except comment.

**Comment by YoshikawaMasashi** [comment]
OK. Done.

**Comment by okuta** [comment]
jenkins, test this please.

**Comment by YoshikawaMasashi** [comment]
I'll write distribution's test.

**Comment by okuta** [comment]
jenkins, test this please.

**Comment by okuta** [comment]
LGTM!

**Pull request merged**